app/ports/models/buildhistory.py

```diff
@@ -99,7 +102,9 @@ def load_database(data):
 
         for buildername in builders:
             # fetch the last build first in order to figure out its number
-            last_build_data = get_data_from_url(get_url_json(buildername, -1))
+            last_build_data, resp_code = get_data_from_url(get_url_json(buildername, -1))
```

Comment:
```suggestion
            last_build_data, _ = get_data_from_url(get_url_json(buildername, -1))
```
It's customary to use `_` for unused variables in Python.

Reply:
I tried to commit this suggestion but got "Invalid email for web commit. Refresh and try again." If anyone is able to commit this suggestion please do, then let's squash and merge the PR.

Reply:
I'm sorry, I misread the contents and for some reason thought that this has already been fixed (while it was still just a suggestion). Anyway, I merged this now, we can still do this cosmetic change in a separate commit, hoping that the site will be fixed now.